### ISSUE TITLE
Create external list of tracks and their respective time window

### ISSUE THREAD

**Issue by Excidion** [Excidion/trackmania_nations_challenge_bot#15]
By giving a "ground truth" the bot does not have to guess the order of tracks from the data. This enables multiple things:
+ Make sure only valid times are taken into account (eg. in case the server hosts the wrong track even for a short amount of time, the bot can correct for it).
+ easier recalling of past events like mentioned in #14 
+ enable flexible timespans for each track or change the "due" date to anything that is not sunday night.

it might make sense to store this list in the DB, so @fanyx input might be needed.

**Comment by Excidion** [comment]
@fanyx will there be a list of tracks and time windows in future versions of your code that the bot could read in?

**Comment by fanyx** [comment]
Yes, i could change the way tracks are loaded into the tmserver using a xaseco plugin to accomodate for this.

For this we could extend the database table 'challenges' to include a CalenderWeek column or two columns 'From' and 'To' that hold the week/days for the track.
Then we could pass the information about the tracks and intended weeks to the telegram bot to fill those fields on startup/check for added tracks or changed week assignments.
The xaseco plugin could read those assignments, find out which track should be online atm and reassign the trackmania server playlist accordingly.

Is this your intention?

**Comment by Excidion** [comment]
How to handle the track selection server-side i leave up to you. But in general to have a place where the tracks and their related timeframes are stored should be the goal.

From my perspective it is enough to have two columns in the table challenges:
+ track
+ starting timestamp **or** calender week (see below)

Unless you want to have breaks within a season (which might be good to have) you only need the starting timestamp (or the ending for that matter) to always identify which track is the current one. 

If we want to restrict ourselfes with changing tracks only on a per-calender-week basis (which I think is the best anyways) we could also just have the calender week as a column. **Maybe we should just start with that and adapt if we ever feel the need to have more flexible timeframes**

**Comment by Excidion** [comment]
**Status** Awating server-side implementation of scheduled tracks and weeks by @fanyx (via yaml file, DB table, whatever, ...). Solution to this will be taylored to fit the implementation.
